sound/soc/sof/mediatek/mt8186/mt8186.c

```diff
@@ -27,6 +27,12 @@
 #include "mt8186.h"
 #include "mt8186-clk.h"
 
+struct snd_sof_mach_info {
```

Comment:
@chunxu-li since there are plans for others to use the same methodology, do you think this struct definition should be moved outsidemaybe?

Reply:
and maybe rename this to be snd_sof_of_mach to be consistent with acpi_mach.

Reply:
@ranj063  Thanks a lot for your advice. In the current sof file structure, which header do you think is more appropriate to put this declaration?

Reply:
@dbaluta 

I think it is ok that move this struct definition to sof-of-dev.h. What do you feel about this idea?